packages/falso/src/lib/credit-card-number.json

```diff
@@ -1,15 +1,15 @@
 {
   "data": [
     {
-      "issuer": "American Express",
```

Comment:
This is a breaking change

Reply:
why do we need this change? issuer seems ok

Reply:
Yeah I listed it as a breaking change above. I've done it because these are not issuers, they are brands/financial services. An Issuer would be the bank (Revolut, Barclays etc)

Reply:
Ok, but did you mention this in the commit message based on the standard version?

Reply:
I said that it's a breaking change but not why https://github.com/ngneat/falso/pull/216/commits/e8173ff3d17a7be4155a2dffec3e916820969a15

Reply:
Cool, that's fine